src/main/java/uk/ac/ebi/eva/pipeline/configuration/readers/VariantAnnotationReaderConfiguration.java

```diff
@@ -27,6 +28,7 @@
 /**
  * Configuration to inject a AnnotationFlatFileReader as a Variant Annotation Reader in the pipeline.
  */
+@Component
```

Comment:
not to be nitpicking, but wouldn't it be good to be consistent? I mean, in the future it might not be clear why is this a `@component`, while the other readers and writers are `@Configuration`